assets/css/admin/forms/_form-group.scss

```
@@ -0,0 +1,6 @@
+.edd-form-group {
+	&__control--is-inline {
+		display: inline-flex;
+		align-items: flex-end;
+	}
+}
```

@robincornett _I_ know what's going on here, but I still find this a bit hard to immediately understand. I wonder if we need to nest these child classes and use the `&` shorthand? It might save some keystrokes typing but I don't think it saves time when understanding how the HTML is constructed based on the CSS.

```suggestion
.edd-form-group__control {

	&--is-inline {
		display: inline-flex;
		align-items: flex-end;
	}
}
```

That seems more obvious and lends itself to eventually being merged in with our [existing styles](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/assets/css/admin/style.scss#L4070-L4081) in the future.